### PR TITLE
Overly verbose bottom sheet label 

### PR DESCRIPTION
**Pull request by owencraston**
### What
- make the accessibility label less verbose
Resolves: https://github.com/CovidShield/mobile/issues/37
### How?
- added the accessibility label as per @svinkle suggestions

### To test
- Load the app into the iOS simulator
- Enable [VoiceOver screen reader](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) on macOS
- Use the Virtual Cursor to navigate to the [collapsed] Sheet control

### Evidence
Screen Shot 2020-06-03 at 7 08 31 PM
Screen Shot 2020-06-03 at 7 09 06 PM

### CODE REVIEW

**Comment by svinkle** [comment]
I realized that the label text I've suggested would be in violation of [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html). The label needs to include the visual text, otherwise voice dictation users will have difficulty activating the control.

Let's adjust the `Button` label to:

> "Covid Shield is {status}, Tap for more information"

/cc @emilybulger

**Comment by emilybulger** [comment]
There are cases where both `COVID Shield is ON` and `Push notifications are OFF` would display (as in screenshot on #37). How do you want to handle that, @svinkle?

The visible copy doesn't change, I don't think.

**Comment by svinkle** [comment]
@emilybulger As long as the first bit of visible text matches the programatic label, it should be fine*. At the very least, [a voice user can use the grid](https://www.youtube.com/watch?v=aqoXFCCTfm4&feature=youtu.be&t=43) to narrow down screen options.

*Still need to test with real people.

File: src/locale/translations/en.json [hunk 1]
```diff
@@ -156,6 +156,8 @@
     "Label": "Developed in partnership with"
   },
   "BottomSheet": {
-    "Collapse": "Close"
+    "Collapse": "Close",
+    "OnStatus": "Covid Shield is on, Tap for more information",
+    "OfStatus": "Covid Shield is off, Tap for more information"
```
Comment (svinkle) [inline src/locale/translations/en.json:161]: Minor typo here: `OfStatus`.

**Pull request merged**